src/responder/common/responder_common.c

```diff
@@ -1341,7 +1341,15 @@ int sss_process_init(TALLOC_CTX *mem_ctx,
 
     ret = confdb_get_bool(rctx->cdb, rctx->confdb_service_path,
                           CONFDB_RESPONDER_CACHE_FIRST,
-                          false, &rctx->cache_first);
+#ifdef BUILD_FILES_PROVIDER
```

Comment:
Hi,

I think it would be more consistent with reading other confdb parametersto define a `CONFDB_RESPONDER_CACHE_FIRST_DEFAULT` in `src/confdb/confdb.h` and move the `#ifdef` there.

bye,
Sumit

Reply:
Thanks, done.